wcivf/apps/people/views.py

```diff
@@ -30,6 +30,11 @@ def get_person(self, queryset=None):
                 # "leaflet_set",
             )
         )
+        queryset = queryset.annotate(
+            previous_party_count=Count(
+                "personpost__previous_party_affiliations"
+            )
+        )
 
```

Comment:
Will still need to use `prefetch_related` here re: https://github.com/DemocracyClub/WhoCanIVoteFor/pull/1126#discussion_r848635026

Reply:
I forgot to write a comment about this: when I add this to the view, it throws an error: 
`Cannot find 'personpost' on Person object, 'personpost__previous_party_affiliations' is an invalid parameter to prefetch_related()`

Reply:
Oh in that case it may need to be `prefetch_related("personpost_set__previous_party_affiliations")`

Reply:
Still not working 😕

Reply:
That worked and agree, much cleaner.